tests/start-cypress-tests.sh

```diff
@@ -30,6 +30,23 @@ fi
 echo "Logging into Kube API server..."
 oc login --server=$CYPRESS_OC_CLUSTER_URL -u $CYPRESS_OC_CLUSTER_USER -p $CYPRESS_OC_CLUSTER_PASS --insecure-skip-tls-verify
 
+if [[ "$CLEAN_UP" == "true" ]]; then
+  echo "Cleaning up test resources. Tests will not run."
+
+  oc delete applications.app.k8s.io --all --all-namespaces
+  oc delete channels.apps.open-cluster-management.io --all --all-namespaces
+  oc delete deployables.apps.open-cluster-management.io --all --all-namespaces
+  oc delete gitopsclusters.apps.open-cluster-management.io --all --all-namespaces
+  oc delete placementrules.apps.open-cluster-management.io --all --all-namespaces
+  oc delete subscriptions.apps.open-cluster-management.io --all --all-namespaces
```

Comment:
Is it OK to delete all subscriptions? They are used to install ACM.

Reply:
Yep since this is run during the canary clean up stage, ACM is either already uninstalled or will be uninstalled.